test/index.js

```diff
@@ -348,6 +348,13 @@ describe('node-precinct', () => {
         assert.ok(!deps.includes('node:nonexistant'));
         assert.deepEqual(deps, ['streams']);
       });
+
+      it("understands quirks around somem modules only being addressable via node: prefix", () => {
+        const deps = precinct.paperwork(path.join(__dirname, 'fixtures', 'requiretest.js'), {
+          includeCore: false
+        });
+        assert.deepEqual(deps, ['test']);
```

Comment:
I would expect this to be true even without the changes to the filtering code. Was that not true?

Reply:
It wasn't on Node 18, because it was filtered out by `includeCore: false`. That's the reason I opened this PR.